client/src/common/helpers.js

```diff
@@ -120,6 +120,45 @@ export const setStateProp = (prop, data, options) => {
   return obj;
 };
 
+export const viewPropsChanged = (nextViewOptions, currentViewOptions) => {
+  return (
+    nextViewOptions.currentPage !== currentViewOptions.currentPage ||
+    nextViewOptions.pageSize !== currentViewOptions.pageSize ||
+    nextViewOptions.sortField !== currentViewOptions.sortField ||
+    nextViewOptions.sortAscending !== currentViewOptions.sortAscending ||
+    nextViewOptions.activeFilters !== currentViewOptions.activeFilters
+  );
+};
+
+export const viewQueryObject = (viewOptions, queryObj) => {
```

Comment:
Optional: naming update... it is what it says, but was a little confused when I first read it, what about changing the name so it has the word "generate" in it, maybe ```generateViewQueryObject```

Reply:
Note: At some we should consider limiting the exports on these helpers to enforce the namespacing use through 
```js
helpers.SOME_HELPER
``` 
or through 
```js
{ SOME_HELPER }
```
Right now it can be all 3, including just using...
```
import SOME_HELPER from ...
```